src/CMakeLists.txt

```diff
@@ -303,14 +303,17 @@ add_library(${LIB_STATIC_Z} STATIC IMPORTED GLOBAL)
 # Setup for python
 #
 find_package(PythonLibs REQUIRED)
-include_directories(${PYTHON_INCLUDE_DIRS})
+include_directories(SYSTEM ${PYTHON_INCLUDE_DIRS})
```

Comment:
It's treated by compiler as a system include (`-isystem`) instead of `-I`,
basically it hides warnings on the code, as we can't fix the libraries'
code.